debian/rules

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/make -f
+# See debhelper(7) (uncomment to enable)
+# output every command that modifies files on the build system.
+#export DH_VERBOSE = 1
+
+
+# see FEATURE AREAS in dpkg-buildflags(1)
+#export DEB_BUILD_MAINT_OPTIONS = hardening=+all
+
+# see ENVIRONMENT in dpkg-buildflags(1)
+# package maintainers to append CFLAGS
+export DEB_CFLAGS_SET=-DDEBUG=0 -O3
+export DEB_CPPFLAGS_SET=-DDEBUG=0 -O3
+export DEB_CXXFLAGS_SET=-DDEBUG=0 -O3
```

Comment:
maybe add parallel build of cmake/make : --parallel or `export CMAKE_BUILD_PARALLEL_LEVEL`
> If this variable is defined empty the native build tool’s default number is used.
 see doc: https://cmake.org/cmake/help/v3.12/manual/cmake.1.html#build-tool-mode